### PR TITLE
5533 - Reduce census block line thickness to match census tracts

### PR DESCRIPTION
**Pull request by bmarchena**
### Summary

Reduces the line thickness of census blocks for improved visibility in [resized default map view](https://github.com/NYCPlanning/labs-factfinder/pull/1010). Thickness now matches census tracts when zoomed in (images below for reference).

#### Tasks/Bug Numbers

 - Resolves AB[#5533](https://dev.azure.com/NYCPlanning/ITD/_sprints/taskboard/Open%20Source%20Engineering/ITD/2022/Q3/Sprint%20O?workitem=5533)
 - Related AB[#5297](https://dev.azure.com/NYCPlanning/ITD/_sprints/taskboard/Open%20Source%20Engineering/ITD/2022/Q3/Sprint%20O?workitem=5297)

Census Blocks:
Screen Shot 2022-07-13 at 11 28 32 AM

Census Tracts:
Screen Shot 2022-07-13 at 11 28 21 AM

### CODE REVIEW

**Review by godfreyyeung — APPROVED** [review]

**Comment by godfreyyeung** [comment]
Wait for @TylerMatteo 's approval too. I think he was looking at this earlier.

**Review by TylerMatteo — APPROVED** [review]

**Comment by TylerMatteo** [comment]
lgtm

**Pull request merged**
